BaseFilter/sections/antiadblock.txt

```diff
@@ -165,7 +165,7 @@ articlix.com,fcc.lc,fantastic-film.online,joinfaucet.com#@#.TextAd
 ! SECTION: BlockAdBlock
 !START: BlockAdBlock
 ! Old and 4.2b - #%#//scriptlet("prevent-bab")
```

Comment:
1. The title of the pull request (you don't have to specify it there) and the commit have wrong task number.
2. Do you need `#%#//scriptlet("prevent-bab")`? It works fine without it.

Reply:
Btw, it does not look like a `bab` detection. More like `antiblock.org g207` for which we have rules.

Reply:
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/201901029-ff473bf4-5b5d-46b2-898d-4bfdfc3802fe.png)

</details><br/>
You have reverted the wrong title. It is not needed.

Reply:
> The title of the pull request (you don't have to specify it there) and the commit have wrong task number.

Okay.

> Btw, it does not look like a bab detection. More like antiblock.org g207 for which we have rules.

I pushed a commit related to g207 anti-adblock at https://github.com/AdguardTeam/AdguardFilters/pull/134956/commits/4031de410d40351b9be2febd152045bb89cdba5a. But, I cannot see blockadblock script on the website too unlike when i check at first time.

Reply:
Don't worry, got it :)